chapter_appendix/math.md

```diff
@@ -220,7 +220,7 @@ The Taylor expansion of function $f$ is given by the infinite sum
 
 $$f(x) = \sum_{n=0}^\infty \frac{f^{(n)}(a)}{n!} (x-a)^n,$$
 
-when it exists. Here, $f^{(n)}$ is the $n$-th derivative of $f$, and $n!$ is the factorial of $n$. For a sufficiently small number $\epsilon$, we can replace $x$ and $a$ with $x+\epsilon$ and $x$ respectively to obtain
+when it exists. Here, $f^{(n)}$ is the $n$th derivative of $f$, and $n!$ is the factorial of $n$. For a sufficiently small number $\epsilon$, we can replace $x$ and $a$ with $x+\epsilon$ and $x$ respectively to obtain
```

Comment:
it may look clearer with -

Reply:
In the previous PR on the math appendix, I had changed this to $n$-th from $n$th for the reason of clarity, but then noticed there are 6 other occurrences in the text that use $x$th so I was changing it back for consistency.

I also did some searching on Stackoverflow for what seems to be the standard in mathematics texts and found that $n$th is in fact more widely used. 

Here's the stack overflow link: https://tex.stackexchange.com/questions/10959/1st-2nd-and-i-th

Let me know what you prefer

Reply:
@smolix Alex, do you think we should go with $n$-th or $n$th? Alternatively, we can do $n^{\text{th}}$